manifests/base/code-server-notebook-imagestream.yaml

```diff
@@ -6,8 +6,8 @@ metadata:
     opendatahub.io/notebook-image: "true"
   annotations:
     opendatahub.io/notebook-image-url: "https://github.com/opendatahub-io/notebooks/tree/main/codeserver"
-    opendatahub.io/notebook-image-name: "Code Server"
-    opendatahub.io/notebook-image-desc: "Code Server workbench image, allows to run Visual Studio Code (VSCode) on a remote server through the browser."
+    opendatahub.io/notebook-image-name: "code-server"
```

Comment:
The naming here deviates from the established standard UI aesthetic. It would be beneficial to align it with the overall design for a more cohesive and visually pleasing interface.